onnxruntime/core/graph/ep_api_types.h

```diff
@@ -216,6 +219,10 @@ struct EpNode : public OrtNode {
   // Helper that gets the execution provider name that this node is assigned to run on.
   const std::string& GetEpName() const;
 
+  // Helper to get the unique name for the 'TENSOR' attribute. Returns empty string if
+  // attribute is not 'TENSOR' type.
+  const std::string GetUniqueTensorAttributeName(const OrtOpAttr* attr) const;
```

Comment:
typically, we don't want to return const by value. it can be copied into a non-const variable anyway. just return `std::string`.

can it be a non-member helper function that is only in ep_abi_types.cc?

Reply:
This function is not needed anymore, since we are not doing caching. I just removed it.